libopendroneid/opendroneid.h

```diff
@@ -145,14 +153,15 @@ typedef struct {
 
 typedef struct {
     ODID_status_t Status;
-    float SpeedNS;            // m/s
-    float SpeedEW;            // m/s
+    float Direction;          // Degrees
```

Comment:
Nice to note degrees take and define what zero degrees is. Is it 0 to 359 where zero is magnetic North? True north? Relative Forward?

Reply:
I will add some clarification.